aptos-move/aptos-gas/src/transaction/mod.rs

```diff
@@ -125,6 +133,41 @@ crate::params::define_gas_parameters!(
             { 5.. => "max_bytes_all_events_per_transaction"},
             10 << 20, // all events from a single transaction are 10MB max
         ],
+        [
+            storage_fee_per_state_slot_create: FeePerSlot,
+            { 7.. => "storage_fee_per_state_slot_create" },
+            50000,
```

Comment:
How did you come up with these numbers?

Reply:
@msmouse has a doc that explains the logic.

Reply:
This appears to be a significant change. How should we update our docs to reflect it? Please let me help. Thanks!

Reply:
@clay-aptos thank you so much for helping! We're still in the process of figuring out the actual value changes, but I anticipate the doc work to cover the following:
1. Explaining the new system to our users, especially how the flat-APT prices are converted back to gas units to maintain compatibility
2. Making it clear that this is part of the effort to reduce cost (certainly not increasing it)
3. What would come in the future (deposits)

Reply:
Adding the *release-notes* tag so we consider highlighting this.